docs/presentation/Content/ContentInstancesOfSpecificClasses.md

```diff
@@ -2,38 +2,161 @@
+
+```ts
+[[include:ContentInstancesOfSpecificClasses.CalculatedProperties.Ruleset]]
+```
+
+![Example of using calculatedProperties attribute](./media/contentInstancesOfSpecificClasses-with-calculatedProperties-attribute.png)
+
+### Attribute: `propertyCategories`
+
+Specifications for custom categories. Simply defining the categories does nothing - they have to be referenced from `PropertySpecification` defined in `propertyOverrides` by `id`. *See [this page](./PropertyCategorySpecification.md) for more details.*
+
+```ts
+[[include:ContentInstancesOfSpecificClasses.PropertyCategories.Ruleset]]
+```
+
+![Example of using propertyCategories attribute](./media/contentInstancesOfSpecificClasses-with-propertyCategories-attribute.png)
+
+### Attribute: `propertyOverrides`
+
+Specifications for various property overrides. *See [this page](./PropertySpecification.md) for more details.*
+
+```ts
+[[include:ContentInstancesOfSpecificClasses.PropertyOverrides.Ruleset]]
+```
+
+![Example of using propertyOverrides attribute](./media/contentInstancesOfSpecificClasses-with-propertyOverrides-attribute.png)
+
+### Attribute: `showImages`
+
+> **Default value:** `false`
+
+Should image IDs be calculated for the returned instances. When `true`, [ImageIdOverride](../customization/ImageIdOverride.md) rules get applied when creating.
+
+### Attribute: `relatedInstances`
+
+Joining [related instances](../Common-Rules/RelatedInstanceSpecification.md) allows performing advanced filtering, grouping, labeling.
+
+```ts
+[[include:ContentInstancesOfSpecificClasses.RelatedInstances.Ruleset]]
+```
+
+  |  | Result                                                                                                                                      |
+  | ------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
+  | SpatialViewDefinitions  | ![Example when doing normal property select](./media/contentInstancesOfSpecificClasses-with-relatedInstances-attribute-3.png) |
+  | ModelSelectors  | ![Example when doing normal property select](./media/contentInstancesOfSpecificClasses-with-relatedInstances-attribute-2.png) |
+  | ModelSelectors filtered by SpatialViewDefinition Yaw    | ![Example when selecting with related properties](./media/contentInstancesOfSpecificClasses-with-relatedInstances-attribute-1.png)     |
+
+## Deprecated Attributes
+
+### Attribute: `handleInstancesPolymorphically`
+
+> **Default value:** `false`
+
+Defines default `classes` polymorphism value. If set to true, all target classes by default will include also include derived class instances. This value is overriden by the `classes`.[`arePolymorphic`](../Common-Rules/MultiSchemaClassesSpecification.md#attribute-arepolymorphic) value.
```

Comment:
```suggestion
Tells whether selecting instances from ECClasses specified in [`classes`](#attribute-classes) and [`excludedClasses`](#attribute-excludedclasses) attributes should be polymorphic or not.

The attribute was replaced by [MultiSchemaClasses.arePolymorphic](../Common-Rules/MultiSchemaClassesSpecification.md#attribute-arepolymorphic) attribute specified individually for each class definition under [`classes`](#attribute-classes) and [`excludedClasses`](#attribute-excludedclasses) attributes. At the moment, to keep backwards compatibility, this attribute acts as a fallback value in case the flag is not specified individually for a class definition.
```